aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -0,0 +1,52 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/Quantizer.h>
+#include <ATen/QTensorImpl.h>
+
+
+namespace at {
+namespace native {
+
+// This is an internal utility function for getting at the QTensorImpl,
+// You should only use this for writing low level
+// setters/getters for QTensorImpl fields; otherwise, you should use
+// the low level setters/getters that were implemented using this.
+// This may be called repeatedly, so make sure it's pretty cheap.
+inline QTensorImpl* get_qtensorimpl(const QTensor& self) {
```

Comment:
Uhhh this looks like a straight up copy paste of what's in aten/src/ATen/Quantizer.cpp

Reply:
Yeah I'm trying to keep this private. Maybe I should put this in header?